main/ertp/glossary/README.md

```diff
@@ -1,25 +1,48 @@
 # ERTP Glossary
 
+## Alleged `<item>`
+`allegedName`, `allegedExtent`, `allegedUnits`
+
+Because there is no way to actually prove the item you are calling or passing through is what it says it is, we prepend `alleged` to these terms to get that sentiment across.
```

Comment:
I would phrase this in terms of someone who receives a payment from someone they don't %100 trust. 

"Because there is no assurance when you receive an object from someone that it is the thing you were hoping to get, we refer to them as alleged until we confirm the provenance."  Maybe provenance isn't the word you watn to use.

Reply:
What about:

"Because there is no assurance when you receive an object from someone that it is the thing you were hoping to get, or vice versa, we initially refer to them as alleged."

Reply:
I like that!